src/Authentication/Helpers/AuthenticationHelper.cs

```diff
@@ -163,5 +163,22 @@ public static AuthenticationMethod GetAuthenticationMethod(string amr)
 
             return Enum.AuthenticationMethod.NotDefined;
         }
+
+        /// <summary>
+        /// Checks if the identifier of the org number is valid
+        /// </summary>
+        /// <param name="vendor">the org number information of the vendor</param>
+        /// <returns>true if the org number identifier is valid</returns>
+        public static bool IsValidOrgIdentifier(IDictionary<string, string> vendor)
```

Comment:
This method does not check if it's a valid org identifier... Only that it starts with `0192:`.